### PR TITLE
Docs frontend tests

### PR DESCRIPTION
**Pull request by jkeane508**
Frontend Ones Implementation and Frontend Tests Docs

### CODE REVIEW

**Comment by jkeane508** [comment]
Currently, the test for frontend torch ones, is failing which I believe might be an issue with num_positional_args. However this is my first draft of the doc based on @rush2406 's work.

File: ivy/functional/frontends/torch/creation_ops.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# local
+import ivy
+
+
+def ones(size, *, out=None, dtype=None, layout=None, device=None, requires_grad=False):
```
Comment (djl11) [inline ivy/functional/frontends/torch/creation_ops.py:5]: we should actually handle `requires_grad`, we can do this by calling `ivy.variable` on the returned value if `requires_grad=True`

File: ivy/functional/frontends/jax/__init__.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
 # flake8: noqa
 from .lax import operators
 from ivy.functional.frontends.jax.lax.operators import *
+from .numpy import functions
```
Comment (djl11) [inline ivy/functional/frontends/jax/__init__.py:4]: I should have made this clear, but we are actually only concerned with adding the `jax.lax` frontend for the time being, and not the `jax.numpy` namespace. If you want to add a similar frontend function for `jax` as well, then you could add [jax.lax.full](https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.full.html#jax.lax.full)

**Review by djl11 — CHANGES_REQUESTED** [review]
a few changes, but otherwise it looks great :)

**Comment by jkeane508** [comment]
Hi,

I've made the changes and full() and is one of the more complex examples which is a good complement to ivy.tan()

However, the test for PyTorch frontend test is not passing and it is not something that I could work out. If I could get a quick review of that with a fresh set of eyes, would be appreciated.

Other than that I'm pretty happy with this document :)

File: ivy/functional/frontends/torch/creation_ops.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    layout=None,
+    device=None,
+    requires_grad=None
+):
+    ret = ivy.full(
+        shape=size, fill_value=fill_value, dtype=dtype, device=device, out=out
+    )
+    if requires_grad:
+        return ivy.variable(ret)
+    else:
```
Comment (djl11) [inline ivy/functional/frontends/torch/creation_ops.py:20]: small detail, but we can actually replace:

```
else:
    return ret
```

with:

```
return ret
```

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py [hunk 4]
```diff
@@ -139,3 +140,62 @@ def test_jax_lax_concat(
...
+        )
+    )
+
+
+@st.composite
+def _fill_value(draw):
+    dtype = draw(_dtypes())[0]
+    if ivy.is_uint_dtype(dtype):
+        return draw(st.integers(0, 5))
+    if ivy.is_int_dtype(dtype):
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_lax_operators.py:163]: `elif`

File: ivy_tests/test_ivy/test_frontends/test_numpy/creation_routines/test_from_shape_or_value.py [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+        )
+    )
+
+
+@st.composite
+def _fill_value(draw):
+    dtype = draw(_dtypes())[0]
+    if ivy.is_uint_dtype(dtype):
+        return draw(st.integers(0, 5))
+    if ivy.is_int_dtype(dtype):
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_numpy/creation_routines/test_from_shape_or_value.py:28]: `elif`

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py [hunk 6]
```diff
@@ -129,3 +130,62 @@ def test_tensorflow_concat(
...
+        )
+    )
+
+
+@st.composite
+def _fill_value(draw):
+    dtype = draw(_dtypes())[0]
+    if ivy.is_uint_dtype(dtype):
+        return draw(st.integers(0, 5))
+    if ivy.is_int_dtype(dtype):
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_tf_functions.py:153]: `elif`

File: ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py [hunk 7]
```diff
@@ -0,0 +1,81 @@
...
+        )
+    )
+
+
+@st.composite
+def _fill_value(draw):
+    dtype = draw(_dtypes())[0]
+    if ivy.is_uint_dtype(dtype):
+        return draw(st.integers(0, 5))
+    if ivy.is_int_dtype(dtype):
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py:28]: `elif`

File: ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py [hunk 8]
```diff
@@ -0,0 +1,81 @@
...
+        return draw(st.integers(-5, 5))
+    return draw(st.floats(-5, 5))
+
+
+@st.composite
+def _requires_grad(draw):
+    dtype = draw(_dtypes())[0]
+    if ivy.is_int_dtype(dtype) or ivy.is_uint_dtype(dtype):
+        return draw(st.just(False))
+    else:
```
Comment (djl11) [inline ivy_tests/test_ivy/test_frontends/test_torch/test_creation_ops.py:38]: see comment above

**Review by djl11 — CHANGES_REQUESTED** [review]
some small changes, could you also please provide a stack trace of the failing `torch` example that you are unable to figure out?

**Pull request merged**